examples/chip-tool/commands/pairing/PairingCommand.h

```diff
@@ -138,6 +140,7 @@ class PairingCommand : public Command,
     uint16_t mDiscriminator;
     uint32_t mSetupPINCode;
     chip::ByteSpan mOperationalDataset;
+    uint8_t mExtendedPanId[8];
```

Comment:
use `kSizeExtendedPanId` for sizing this.

Reply:
Done.